include/ppx/knob.h

```diff
@@ -314,6 +316,67 @@ class KnobDropdown final
     std::vector<T> mChoices;
 };
 
+// KnobConstant is intended for parameters that cannot be adjusted when the application is run
+// They will not be displayed in the UI and cannot have their values changed by the application
+template <typename T>
+class KnobConstant final
```

Comment:
Completely up to you, but how about naming these KnobFlag? That way it is clear what they are used for, unless you think there could be another use case for these.